### ISSUE TITLE
Compatibility with Particle Tweaks

### ISSUE THREAD

**Issue by xen2ken** [PigCart/particle-rain#53]
It would be nice to see this mod being compatible with Particle Tweaks, two awesome mods that will change minecraft look so much.

**Comment by PigCart** [comment]
particles already fade in and out what "compat" would i be adding

**Comment by xen2ken** [comment]
Yes, I know that. But I meant something different, Particle Tweaks improves all minecraft particles, but when I am trying to launch those 2 mods, minecraft crashes. You're right, Particle Rain has that animation, but it would be nice to see those mods work together, because Particle Rain + Smooth all minecraft particles is just an awesome combo.

**Comment by xen2ken** [comment]
[tl-logger4675590904516866129.txt](https://github.com/user-attachments/files/17381035/tl-logger4675590904516866129.txt)

**Comment by PigCart** [comment]
ah, this is a duplicate of #48 
you will have to use an older version of particle rain or particle tweaks

**Comment by xen2ken** [comment]
Just installed particle tweaks 1.1.1, everything works perfectly! Thank you so much. ( Maybe put this issue in mod description, to let ppl know :3 )

Thank you

**Comment by xen2ken** [comment]
Your mod is such a beautiful creation, it uses minecraft vanilla textures, amazing choice! It enhances minecraft's visual aspect in so good way, I lovey your mod now, imo it's a must have in minecraft mod packs :з

![2024-10-15_13 08 00](https://github.com/user-attachments/assets/efe13448-4dad-4de7-81f1-ec525b6ab020)
